modules/rules_cc/0.0.9/source.json

```diff
@@ -0,0 +1,9 @@
+{
+    "integrity": "sha256-IDeHW5pEVtzkp50RKorohbvEqtlo5lh9ym5k86CQDN8=",
+    "strip_prefix": "rules_cc-0.0.9",
+    "url": "https://github.com/bazelbuild/rules_cc/archive/refs/tags/0.0.9.tar.gz",
```

Comment:
@buildbreaker2021 Could you replace this with https://github.com/bazelbuild/rules_cc/releases/download/0.0.9/rules_cc-0.0.9.tar.gz and also update the `integrity` hash?

Reply:
PTAL at https://github.com/bazelbuild/bazel-central-registry/pull/942 I have created a fresh release, everything should be fine there(I hope at least).